datadog-service/0.15.0/README.md

```diff
@@ -0,0 +1,198 @@
+# datadog-service
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/keptn-sandbox/datadog-service)
+[![Go Report Card](https://goreportcard.com/badge/github.com/keptn-sandbox/datadog-service)](https://goreportcard.com/report/github.com/keptn-sandbox/datadog-service)
+
+This implements a datadog-service for Keptn. If you want to learn more about Keptn visit us on [keptn.sh](https://keptn.sh)
+
+Check the issue on the main repo for more info: https://github.com/keptn/keptn/issues/2652
+
+
+## Quickstart
+If you are on Mac or Linux, you can use [examples/kup.sh](https://raw.githubusercontent.com/keptn-sandbox/datadog-service/release-0.15.0/examples/kup.sh). This script creates a local minikube cluster, installs Keptn, Istio, Datadog and the Datadog integration for Keptn (check the script for pre-requisites). 
+
+To use the script,
+```bash
+$ export DD_API_KEY="<your-datadog-api-key>" DD_APP_KEY="<your-datadog-app-key>" DD_SITE="datadoghq.com" 
+$ examples/kup.sh
+```
+Check [the official docs](https://docs.datadoghq.com/account_management/api-app-keys/) for how to create Datadog API key and Application key
+
+Note: Application keys get the same permissions as you. You might want to narrow down the permissions (datadog-service only reads the metrics from the API)
+
+## If you already have a Keptn cluster running
+1. Install datadog
+```bash
+$ export DD_API_KEY="<your-datadog-api-key>" DD_APP_KEY="<your-datadog-app-key>" DD_SITE="datadoghq.com" 
+$ helm install datadog --set datadog.apiKey=${DD_API_KEY} datadog/datadog --set datadog.appKey=${DD_APP_KEY} --set datadog.site=${DD_SITE} --set clusterAgent.enabled=true --set clusterAgent.metricsProvider.enabled=true --set clusterAgent.createPodDisruptionBudget=true --set clusterAgent.replicas=2
+
+```
+2. Install Keptn datadog-service
+```bash
+$ export DD_API_KEY="<your-datadog-api-key>" DD_APP_KEY="<your-datadog-app-key>" DD_SITE="datadoghq.com" 
+# cd datadog-service
+$ helm install datadog-service ./helm --set datadogservice.ddApikey=${DD_API_KEY} --set datadogservice.ddAppKey=${DD_APP_KEY} --set datadogservice.ddSite=${DD_SITE}
+```
+
+3. Add SLI and SLO
```

Comment:
```suggestion
3. Add  the SLIs and SLOs to be used for evaluation
```
Do you want to say anything about customizing these, especially the SLO's?

Reply:
I think the SLOs are standard affair i.e., nothing changes from other SLI providers. I will add a link to the examples in the repository. Do you think I should add something else as well?

Reply:
Added example links here: https://github.com/keptn-contrib/artifacthub/pull/141/files#diff-f753100bbf2810b9459b87be72f6913e543e5a409fa0f3698de621ed115cdbf7R47

Reply:
@StackScribe in general we always refer to "SLIs and SLOs" (as in: Service Level Indicators and Service Level Objectives), not "the SLIs and SLOs", unless you explictly mention "the SLI file" or "the SLO file".

Reply:
I think it makes sense to keep the line as is then since I am adding only 1 SLI and 1 SLO